server_addon/houdini/server/settings/publish.py

```diff
@@ -45,6 +45,39 @@ class CollectLocalRenderInstancesModel(BaseSettingsModel):
     )
 
 
+def product_types_enum():
+    return [
+        {"value": "camera", "label": "Camera (Abc)"},
+        {"value": "pointcache", "label": "PointCache (Abc)/PointCache (Bgeo)"},
+        {"value": "review", "label": "Review"},
+        {"value": "staticMesh", "label": "Static Mesh (FBX)"},
+        {"value": "usd", "label": "USD (experimental)"},
+        {"value": "vdbcache", "label": "VDB Cache"},
+        {"value": "imagesequence", "label": "Composite (Image Sequence)"},
+        {"value": "ass", "label": "Arnold ASS"},
+        {"value": "arnold_rop", "label": "Arnold ROP"},
+        {"value": "mantra_rop", "label": "Mantra ROP"},
+        {"value": "redshiftproxy", "label": "Redshift Proxy"},
+        {"value": "redshift_rop", "label": "Redshift ROP"},
+        {"value": "karma_rop", "label": "Karma ROP"},
+        {"value": "vray_rop", "label": "VRay ROP"},
+        {"value": "model", "label": "Model"},
+    ]
```

Comment:
@MustafaJafar with your other comment that this 'doesn't work correctly for all options of the render options' is it still valid to include the render rops in this list for CollectFilesForCleanup?

Reply:
the logic of `CollectFilesForCleaningUp`, it works before `CollectLocalRenderInstances` 
which means instances with `render` doesn't exist yet.
Personally, I think it is easier and it allows to select whether to clean render from mantra or render from karma and etc.

Reply:
Not sure I understand. Is there something you wish to change now still? or should it be good as is?

The way I understand, render instances don't exist yet - so it doesn't work? So do the selection entries here still make sense, or?

Reply:
I believe it's good as it is.

---

Render instances don't exist yet at that point of execution. 
But, they have nothing to do with the existence of files because extractors works on the original instance. e.g. `mantra_rop`.   

`CollectFilesForCleaningUp` plugins works in a similar fashion to extractors, as it works on the original instance data.

Reply:
My brain can't cope with the explanation today :D - anyway if it's "good as is" I'll mark this conversation resolved and assume it doesn't break anything for the render products.

Reply:
> My brain can't cope with the explanation today :D - anyway if it's "good as is" I'll mark this conversation resolved and assume it doesn't break anything for the render products.

It doesn't hit the error although I somehow doubt if it should be really there. (Maybe it should be)

Reply:
In contrast, My brain can't explain today :D
I wish if you could explain it furtherly but I can't find any words atm.